tools/image_processing/imagej2/imagej2_watershed_binary/imagej2_watershed_binary.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_watershed_binary" name="Watershed segmentation" version="1.0.0">
+    <description>of binary image with ImageJ2</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_20141125_requirements" />
```

Comment:
I don't think they will produce the same results.
At some point we release this version to MTS. All these tool will depend on version 20141125 and get an installable revision. After a few month we will upgrade the dependency (add the test files ;)) and upload everything. We will get a new installable revision and everything will be good :)